docs/quick-install.md

```diff
@@ -99,6 +99,7 @@ To install the Smart Agent on a single Windows host, enter:
 & {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "<TOKEN>"; ingest_url = "https://ingest.<REALM>.signalfx.com"; api_url = "https://api.<REALM>.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
 ```
 
+
```

Comment:
Is the change only supposed to be one extra blank line?

Reply:
The change was to replace the script for Windows. 

Originally, it was: 

& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"};; 
ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}

And now, it should be: 

& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "<TOKEN>"; ingest_url = "https://ingest.<REALM>.signalfx.com"; api_url = "https://api.<REALM>.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}

Reply:
But, the code may be incorrectly written because in the UI, realm and access token isn't automatically populated.(See screenshot.) 
<img width="1378" alt="Screen Shot 2019-10-09 at 11 39 37 AM" src="https://user-images.githubusercontent.com/54906773/66498769-9ce3b300-ea8c-11e9-8eb5-8b0129393c9e.png">

Reply:
ok, that command was not changed in this PR though.

Reply:
To confirm, "app.signalfx.com" is prod, right? If so, then it appears that this text was already updated, which would explain why my copy / paste did not work register as a change in the document. But, this brings up another issue: shouldn't the token and realm be populated? If you look at the screenshot, it's not.

Reply:
looks like the "magic" words in signalview are
  const tokenPlaceholder = new RegExp(/YOUR_SIGNALFX_API_TOKEN/g);
  const orgIdPlaceholder = new RegExp(/YOUR_SIGNALFX_ORG_ID/g);
  const realmPlaceholder = new RegExp(/YOUR_SIGNALFX_REALM/g);

Reply:
The Linux version uses:  YOUR_SIGNALFX_REALM and YOUR_SIGNALFX_API_TOKEN

Can we just use that for Windows?

Reply:
Yes, it should be the same thing.

Reply:
I just made the updates in the docs. Let me close this PR and then make a new one.